src/main/java/org/monarchinitiative/fhir2hpo/loinc/DefaultLoinc2HpoAnnotation.java

```diff
@@ -0,0 +1,126 @@
+package org.monarchinitiative.fhir2hpo.loinc;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.hl7.fhir.dstu3.model.Observation;
+import org.monarchinitiative.fhir2hpo.codesystems.CodeableConceptAnalyzer;
+import org.monarchinitiative.fhir2hpo.codesystems.Loinc2HpoCodedValue;
+import org.monarchinitiative.fhir2hpo.hpo.HpoTermWithNegation;
+import org.monarchinitiative.fhir2hpo.loinc.exception.ConflictingInternalCodesException;
+import org.monarchinitiative.fhir2hpo.loinc.exception.UnmappedCodeableConceptException;
+import org.monarchinitiative.fhir2hpo.loinc.exception.UnmappedInternalCodeException;
+
+/**
+ * This represents the default annotation implementation where a single observation is parsed
+ * to determine the HPO mapping. It will try the following approaches in order:
+ * - Find HPO through interpretation CodeableConcept
+ * - Find HPO through value quantity
+ * - Find HPO through value CodeableConcept
+ * 
+ * @author yateam
+ *
+ */
+public class DefaultLoinc2HpoAnnotation implements Loinc2HpoAnnotation {
+	
+    private LoincId loincId;
+	private LoincScale loincScale;
```

Comment:
This file may need some minor formatting.

Reply:
Yes, several of the files have formatting issues that seem to be a result of Eclipse and Github not playing nicely together. Once I choose to format the file in Eclipse, most of these issues go away. I will do that as part of a separate pull request but didn't want to muddy this with extraneous files.